### ISSUE TITLE
Add a "support preloading" use-case

### ISSUE THREAD

**Issue by yoavweiss** [ResponsiveImagesCG/ri-usecases#31]
Going over the use-cases a few days ago, I've noticed that we left out one major use-case: That the solution will have high-performance.

I propose to add a use-case claiming that the solution should be (at least as its default) preloader-friendly.

Thoughts?

**Comment by Wilto** [comment]
Hm. Mentioning the preloader might be too implementation-specific for the spec. It couldn’t hurt; maybe a “non-normative” note somewhere?

**Comment by yoavweiss** [comment]
I'm not sure saying the solution should be preloaderable (yeah, that's not a word) is out of limits and needs to be placed in a "non-normative" section. Implementations can still decide whether to preload or not as they will.

**Comment by anselmh** [comment]
Well, I don't know if that is a "use-case". 
I mean, the use-case document is not about implementation, right? Shouldn't this be an "non-normative" notice in the specification?

**Comment by igrigorik** [comment]
tl;dr.. +1

Performance is a use case, because performance is a feature, just like any other.

@pbakaus made this very point at a recent event (Mobile Summit at I/O), and I couldn't agree with him more: W3C specs and standards should have performance criteria attached to them. There are great examples of this in other industries - e.g., all 3GPP / 3GPP2 specs specify performance targets, etc. It's about time we start paying attention to performance also.

**Comment by anselmh** [comment]
Thanks. So we still need to figure some points out here before adding this section:
- Should it be marked as 'non-normative' or not?
- Should it be in specification? (IMO yes)
- Should it additionally be mentioned in use-cases document? (IMO no because this document is mostly for developers but have no strong opinion on that)

**Comment by yoavweiss** [comment]
I don't think this should be mentioned in the picture spec, because picture (and srcset) are both solutions that are (more or less) preloader-friendly (based on the initial viewport).
There are proposed solutions that suggest waiting for layout. In my book these solutions are not valid ones for the default case (they might be valid for some edge cases), because they are not as performant as they can be, but without a "preloader-friendly" use-case, they satisfy all the use-cases.
This is the reason I think we need this as a _real_ use-case (i.e. not in a "non-normative" jail).

**Comment by yoavweiss** [comment]
Closed by https://github.com/ResponsiveImagesCG/ri-usecases/pull/34
